src/types/config.ts

```diff
@@ -4,6 +4,8 @@ export interface BasicStakingConfig {
 	BUY_LINK?: string;
 	farmStartTimeMS?: number;
 	icon?: string;
+	active?: boolean;
```

Comment:
may we should decide between this and the active property in the simpleStakingConfig,
they are same!